_docs/reference/cli/namespace.md

```diff
@@ -7,41 +7,37 @@ module: reference/cli/namespace
 
 # Namespaces
 
-Namespaces help different projects or teams share a StorageOS cluster. No
-namespaces are created by default, and users can have any number of namespaces.
```

Comment:
I think we should have a link to the concept in the other docs area, in case they went directly to the CLI docs and need more background - I this would apply to most of the CLI commands.